experiments/AtmosLES/channel-flow.jl

```diff
@@ -0,0 +1,190 @@
+using ClimateMachine
```

Comment:
I think we're trying to avoid `-` in filenames (for Windows?) so please rename to `channel_flow.jl`.